### PR TITLE
Declarative hydra config with PRs

### PR DESCRIPTION
**Pull request by ulrikstrid**
https://hydra.strid.ninja/project/nix-overlays

### CODE REVIEW

File: ci/hydra_spec.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "keepnr": 10,
+  "inputs": {
+    "nixpkgs": {
+      "type": "git",
+      "value": "https://github.com/nixos/nixpkgs nixos-unstable",
+      "emailresponsible": false
+    },
+    "src": {
+      "type": "git",
+      "value": "https://github.com/anmonteiro/nix-overlays ulrikstrid-hydra-declarative-conf",
```
Comment (ulrikstrid) [inline ci/hydra_spec.json:20]: This should be changed before we merge

File: ci/simple-pr-dummy.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (ulrikstrid) [inline ci/simple-pr-dummy.json:1]: Maybe we should just add a correct file but everything except the jst 0.14 PR?

File: ci/hydra_spec.json (hunk 1 above)
Comment (ulrikstrid) [inline ci/hydra_spec.json:20]:
```suggestion
      "value": "https://github.com/anmonteiro/nix-overlays",
```

File: ci/hydra_spec.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  "description": "nix-overlays tests",
+  "nixexprinput": "src",
+  "nixexprpath": "ci/hydra_jobs.nix",
+  "checkinterval": 300,
+  "schedulingshares": 100,
+  "enableemail": false,
+  "emailoverride": "",
+  "keepnr": 10,
+  "inputs": {
+    "nixpkgs": {
```
Comment (anmonteiro) [inline ci/hydra_spec.json:13]: Are you sure we need this? Supposedly we'll be building against a specific hash of nixpkgs

**Review by anmonteiro — APPROVED** [review]
don't forget to commit the suggestion

**Pull request merged**